lib/js/components/Form/InputField/InputField.vue

```diff
@@ -128,22 +128,13 @@
 
 <script lang="ts" setup>
 import { computed, InputHTMLAttributes } from 'vue';
-import FormField, { FORM_FIELD_STATES, FormFieldProps, FormFieldSlots } from '../FormField';
-import Icon, { ICON_SIZES, IconItem } from '../../Icons/Icon';
+import FormField, { FORM_FIELD_STATES } from '../FormField';
+import Icon, { ICON_SIZES } from '../../Icons/Icon';
 import { extractFormFieldProps } from '../FormField/FormField.utils';
+import { InputFieldProps, InputFieldSlots } from './InputField.types';
 
-interface Props extends FormFieldProps {
-	inputProps?: InputHTMLAttributes;
-	leftIcon?: IconItem | null;
-	suffixText?: string;
-}
-
-interface Slots extends Omit<FormFieldSlots, 'field'> {
-	action?: () => any;
-}
-
-const { inputProps, leftIcon, suffixText, ...rest } = defineProps<Props>();
-defineSlots<Slots>();
+const { inputProps, leftIcon, suffixText, ...rest } = defineProps<InputFieldProps>();
```

Comment:
IMO changes like this in InputForm should be added in InputForm PR, here it obscures the readability of PR - especially since this PR is based to PR with InputForm changes